api/src/pcapi/routes/native/v1/serialization/offers.py

```diff
@@ -80,7 +82,13 @@ def _get_non_scrappable_activation_code(stock: Stock) -> dict | None:
     def from_orm(cls, stock: Stock) -> "OfferStockResponse":
         stock.cancellation_limit_datetime = cls._get_cancellation_limit_datetime(stock)
         stock.activationCode = cls._get_non_scrappable_activation_code(stock)
-        return super().from_orm(stock)
+        stock_response = super().from_orm(stock)
+        price_category = getattr(stock, "priceCategory", None)
+        if price_category:
+            stock_response.priceCategoryLabel = price_category.priceCategoryLabel.label
+        else:
+            stock_response.priceCategoryLabel = None
+        return stock_response
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏
On peut garder la même syntaxe que dans bookings/api.py:

```python
stock_response.priceCategoryLabel=(
        price_category.priceCategoryLabel.label if price_category else None
),
```